01.fit_a_line/README.md

```diff
@@ -185,25 +184,24 @@ Moreover, an event handler is provided to print the training progress:
 
 ```python
 # Specify the directory path to save the parameters
-params_folder = "fit_a_line.inference.model"
+params_dirname = "fit_a_line.inference.model"
 
 # Plot data
 from paddle.v2.plot import Ploter
 train_title = "Train cost"
 test_title = "Test cost"
 plot_cost = Ploter(train_title, test_title)
+
 step = 0
 
 # event_handler to print training and testing info
-def event_handler(event):
+def event_handler_plot(event):
     global step
```

Comment:
Here we still need a global step because `event.step` only goes 0-20 in one EndStepEvent. We need the step to keep growing. Otherwise the plot would look weird like zig-zag.